### PR TITLE
EVG-12882 list parameters

### PR DESCRIPTION
**Pull request by ablack12**
Output example:
```
Name      Default       Description
----      -------       -----------
my_param  hello world!  something to test parameters
```

### CODE REVIEW

**Comment by ablack12** [comment]
lint patch https://spruce.mongodb.com/version/5f5f9fbd2a60ed619ec365d0/tasks

**Comment by ablack12** [comment]
lint tested locally, passing except for the currently failing thing in the rest/route test, which I think I fixed in the other PR

File: rest/route/project.go [hunk 1]
```diff
@@ -802,3 +802,44 @@ func (h *GetProjectAliasResultsHandler) Run(ctx context.Context) gimlet.Responde
...
+	p, err := dbModel.FindLastKnownGoodProject(h.projectId)
+	if err != nil {
+		return gimlet.NewJSONInternalErrorResponse(errors.Wrapf(err,
+			"error finding project config for project '%s'", h.projectId))
+	}
+	if p == nil {
+		return gimlet.NewJSONErrorResponse(errors.Errorf("project '%s' not found", h.projectId))
+	}
+
+	return gimlet.NewJSONResponse(p.Parameters)
```
Comment (john-m-liu) [inline rest/route/project.go:844]: there should be a rest model shouldn't there?

**Review by john-m-liu — CHANGES_REQUESTED** [review]

File: rest/route/project.go (hunk 1 above)
Comment (ablack12) [reply rest/route/project.go:844]: you right

**Review by john-m-liu — APPROVED** [review]

**Pull request merged**